tests/test_random_beacon_history.cdc

```diff
@@ -69,6 +70,15 @@ fun testRecordRandomSource() {
         admin
     )
     Test.expect(txResult, Test.beSucceeded())
+    // get backfiller max entries 
```

Comment:
The backfiller max entries value has a dedicated test `testBackfillerMaxEntryPerCall`. Does this case check a specific behaviour non tested in `testBackfillerMaxEntryPerCall` ?